modules/apps/frontend-theme-valencia/src/js/main.js

```diff
@@ -0,0 +1,87 @@
+AUI().ready(
+
+	/*
+	This function gets loaded when all the HTML, not including the portlets, is
+	loaded.
+	*/
+
+	function() {
+
+		$('.carousel').carousel();
+
+		$('.dropdown-toggle').dropdown('toggle');
+
+		var transparent = true;
+
+		if ($('.navbar-color-on-scroll').length !== 0) {
+			$(window).on( 'scroll', debounce(function () {
+				if (!transparent) {
+					transparent = true;
+
+					$('.navbar-color-on-scroll').addClass('navbar-transparent');
+				}
+
+
+				if ($(document).scrollTop() > 100 && transparent) {
+					transparent = false;
+
+					$('.navbar-color-on-scroll').removeClass('navbar-transparent');
+				}
+			}, 17));
+		}
+
+		function debounce(func, wait, immediate) {
+			var timeout;
+
+			return function () {
+				var args = arguments;
+				var instance = this;
+
+				clearTimeout(timeout);
+
+				timeout = setTimeout(
+					function() {
+						timeout = null;
+
+						if (!immediate) {
+							func.apply(instance, args);
+						}
+					}, wait)
+				;
+
+				if (immediate && !timeout) {
+					func.apply(instance, args);
+				}
+			};
+		}
+		
```

Comment:
Clarity: Should the end result be:
`}};}`
on one line? or is the problem with extra tabs on line 57?